src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -32,6 +33,7 @@
 
     // NOTE: keeping the driving source of variants separate from other, supplementary FeatureInputs in our FeatureManager in GATKTool
     private FeatureDataSource<VariantContext> drivingVariants;
+    private FeatureInput<? extends Feature> drivingVariantFeatureInput;
```

Comment:
Rename to `drivingVariantsFeatureInput` for consistency with `drivingVariants` above.